Makefile

```diff
@@ -23,13 +23,15 @@ test:
 	go install -v gopkg.in/reform.v1/internal/test/models
 	go test -i -v
 	go install -v github.com/kisielk/errcheck \
-					github.com/golang/lint/golint \
+					github.com/alecthomas/gometalinter \
 					github.com/AlekSi/goveralls
+	gometalinter --install
 
 check: test
 	go vet ./...
 	-errcheck ./...
 	golint ./...
```

Comment:
I don't think there is a need to invoke vet, errcheck, golint separately – gometalinter should do it.